src/cider/nrepl/middleware/track_state.clj

```diff
@@ -19,27 +20,6 @@
                        (catch Exception e nil)))
 
 ;;; Auxiliary
-(defn update-vals
-  "Update the keys of map `m` via the function `f`."
-  [f m]
-  (reduce (fn [acc [k v]]
-            (assoc acc k (f v)))
-          {} m))
-
-(def relevant-meta-keys
-  "Metadata keys that are useful to us.
-  This is used so that we don't crowd the ns cache with useless or
-  redudant information, such as :name and :ns."
-  [:indent :deprecated :macro :arglists :test
-   :cider.nrepl.middleware.util.instrument/breakfunction
-   :style/indent :clojure.tools.trace/traced])
-
-(defn relevant-meta
```

Comment:
I had to move these out, because this ns already depends on `cider.nrepl.middleware.ns` and we want the `relevant-meta` fn there, causing a cyclic dependency.